### PR TITLE
LIBS-332 Make hmac-sha256 signed upload signature

### PR DESCRIPTION
**Pull request by yarikbratashchuk**
## Description

Make hmac-sha256 signed upload signature
Set signed upload ttl to 60 seconds

### CODE REVIEW

File: ucare/config.go [hunk 1]
```diff
@@ -13,14 +13,11 @@ const (
 	simpleAuthScheme    = "Uploadcare.Simple"
 	signBasedAuthScheme = "Uploadcare"
 	dateHeaderFormat    = time.RFC1123
+
+	signedUploadTTL = 60 * time.Second
```
Comment (homm) [inline ucare/config.go:17]: It should be configurable (probably in another PR)

File: ucare/auth_test.go [hunk 2]
```diff
@@ -47,10 +47,10 @@ func testSignBasedRESTAPIAuthParam(t *testing.T) {
 	assert.Equal(t, expected, authParam)
 }
 
-func testSignBasedUploadAPIAuthParam(t *testing.T) {
+func TestSignBasedUploadAPIAuthParam(t *testing.T) {
 	secret := "project_secret_key"
 	now := int64(1454903856)
-	expected := "46f70d2b4fb6196daeb2c16bf44a7f1e"
+	expected := "d39a461d41f607338abffee5f31da4d4e46535651c87346e76906bf75c064d47"
```
Comment (homm) [inline ucare/auth_test.go:53]: implementation verified:

```python
In [1]: def _signed_upload_signature(secret, expire): 
   ...:     k, m = str(secret).encode(), str(expire).encode() 
   ...:     return hmac.new(k, m, hashlib.sha256).hexdigest() 
   ...:                                                                                                                         

In [2]: import hmac, hashlib                                                                                                    

In [3]: _signed_upload_signature('project_secret_key', 1454903856)                                                              
Out[3]: 'd39a461d41f607338abffee5f31da4d4e46535651c87346e76906bf75c064d47'
```

File: ucare/config.go (hunk 1 above)
Comment (yarikbratashchuk) [reply ucare/config.go:17]: not configurable in pyuploadcare, but probably you are right
but yeah, separate PR, not now

File: ucare/auth_test.go (hunk 2 above)
Comment (yarikbratashchuk) [reply ucare/auth_test.go:53]: cool

**Pull request merged**